frontend/packages/erd-core/src/components/ERDRenderer/ERDRenderer.tsx

```diff
@@ -30,23 +30,23 @@ export const ERDRenderer: FC = () => {
     <div className={styles.wrapper}>
       <AppBar />
       <SidebarProvider defaultOpen={defaultOpen}>
-        <div className={styles.mainWrapper}>
-          <LeftPane />
-          <main className={styles.main}>
-            <div className={styles.triggerWrapper}>
-              <SidebarTrigger />
-            </div>
-            <TableDetailDrawerRoot>
-              <ReactFlowProvider>
+        <ReactFlowProvider>
```

Comment:
The ReactFlowProvider has been moved out of the LeftPane to retrieve ReactFlow information from the LeftPane.